my_main.py

```diff
@@ -0,0 +1,67 @@
+# HMMMM what should i do
+# I think book-reading should be an easy one. Or a math computation of some kind.
+# Inputs: number
+# Outputs: all possible ways of adding digits together, printed out
+
+def print_all(n):
+    s = str(n)
+    for d in s:
+        # I should make some tea. Black tea? To stay awake. I need to make good progress by tomorrow
+        print(d)
+        # Let's try an example
+        # Suppose my number is 4371730
+        # Then the final output needs to be...
+        # 4371730:
+        # 4 + 3 + 7 + 1 + 7 + 3 + 0 = 25 =>
+        # 2 + 5 = 7
+        # 4|371730:
+        # 4 = 4, 3 + 7 + 1 + 7 + 3 + 0 = 21 =>
+        # 4 = 4, 2 + 1 = 3 =>
+        # 4 + 3 = 7
+        # 43|71730: 4 + 3 = 7, 7 + 1 + 7 + 3 + 0 = 18 => 7 = 7, 1 + 8 = 9 => 7 + 9 = 16 => 1 + 6 = 7
+        # 437|1730: 4 + 3 + 7 = 14, 1 + 7 + 3 + 0 = 11 => 1 + 4 = 5, 1 + 1 = 2 => 5 + 2 = 7
+        # 4371|730: 4 + 3 + 7 + 1 = 15, 7 + 3 + 0 = 10 => 1 + 5 = 6, 1 + 0 = 1 => 6 + 1 = 7
+        # It's cool that it's always 7. The proof is actually in the fact that we work with a decimal system but this is a neat little demonstration that it does work
+
+        # It smells. Suspiciously so
+        # Oh no, the kettle!!
+        raise Exception
+
```

Comment:
Clarification: what's the purpose of each of the following functions?

Reply:
This is the kind of info that should be included in a docstring (which are sorely lacking in this code). Good question! Would you still like a walkthrough on what each does or is this for review purposes? Because this is a totally legit thing to ask when you are reviewing somebodies work